<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bomberman</class>
 <widget class="QMainWindow" name="Bomberman">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="menu" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>551</width>
      <height>301</height>
     </rect>
    </property>
    <widget class="QLabel" name="title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>541</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>60</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bomberman</string>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="optionsButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="playMenu" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>581</width>
      <height>341</height>
     </rect>
    </property>
    <widget class="QLabel" name="playTitle">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>251</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>55</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PLAY</string>
     </property>
    </widget>
    <widget class="QPushButton" name="onlineButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Online</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backPlayButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>210</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="optionsMenu" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>701</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QLabel" name="optionsTitle">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>341</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>55</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backOptionsButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditUsername">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeUsernameButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="onlineMenu" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>661</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QListWidget" name="listRooms">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>281</width>
       <height>391</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="labelRooms">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>631</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Created rooms               Create room</string>
     </property>
    </widget>
    <widget class="QPushButton" name="joinButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>440</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Join</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditMaxPlayers">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>110</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="labelMaxPlayers">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>110</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Max Players</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backOnlineButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>440</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditRoomName">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>60</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="labelRoomName">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Room name</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>-10</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>USERNAME</string>
    </property>
   </widget>
   <widget class="QWidget" name="connectMenu" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>551</height>
     </rect>
    </property>
    <widget class="QTextEdit" name="textEditAddress">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>110</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditPort">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>190</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="labelPort">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPort_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Server Address</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connectButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>250</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backConnectButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>310</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditConnectedInfo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>211</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="roomMenu" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>661</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelRoomName2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>-10</y>
       <width>421</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Room</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startGameButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>341</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start game</string>
     </property>
    </widget>
    <widget class="QPushButton" name="leaveRoomButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Leave room</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlayers">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Players</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gameEndMenu" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>541</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelWinInfo">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>120</y>
       <width>421</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Players scores:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backGameEndButton">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>330</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelScore">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>341</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Game ended!</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelWinInfo_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>731</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Yellow           Blue           Green           Red</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Player1score</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Player2score</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>260</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Player3score</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>260</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Player4score</string>
     </property>
    </widget>
   </widget>
   <zorder>menu</zorder>
   <zorder>username</zorder>
   <zorder>connectMenu</zorder>
   <zorder>onlineMenu</zorder>
   <zorder>roomMenu</zorder>
   <zorder>optionsMenu</zorder>
   <zorder>playMenu</zorder>
   <zorder>gameEndMenu</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
